Jenkinsfile

```diff
@@ -65,100 +81,146 @@ pipeline {
             }
         }
 
-        // Variables are not supported in declarative pipeline.
-        // Possible workaround: store SpawnedTestbedVMNames in stashed file.
-        // def SpawnedTestbedVMNames = ''
-
-        // NOTE: Currently nesting multiple stages in lock directive is unsupported
-        stage('Provision & Deploy & Test') {
-            agent none
+        stage('Lock') {
+            agent { label 'ansible' }
```

Comment:
Should we use `renter` instead of `ansible` here?

Reply:
Yes, but after PR with creating and deploying renter VM. For now we have to use `ansible`